quickstart/php/sms/example-7/send-sms-during-call.5.x.php

```diff
@@ -15,7 +15,7 @@
 // otherwise, consider them just another monkey
 $name = $people[$_REQUEST['From']] ?: 'Monkey';
 
-$response = new TwiML();
+$response = new VoiceResponse();
 $response->say("Hello {$name}");
 $response->message("{$name}, thanks for the call!");
```

Comment:
Can't `say` and `message`. Should the second one be `sms`?

Reply:
Yeah I saw that too, will fix